src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -0,0 +1,180 @@
+            if (string.IsNullOrEmpty(connectionString))
+            {
+                throw new CliException("Unable to retrieve storage connection string.");
+            }
+
+            var settings = new AzureStorageOrchestrationServiceSettings
+            {
+                TaskHubName = TaskHubName,
+                StorageConnectionString = connectionString,
+            };
+
+            _orchestrationService = new AzureStorageOrchestrationService(settings);
+            _client = new TaskHubClient(_orchestrationService);
+        }
+
+        public async Task DeleteHistory()
+        {
+            await _orchestrationService.DeleteAsync();
+            ColoredConsole.Write(Green("History and instance tables and queues successfully deleted."));
+        }
+
+        public async Task GetHistory(string instanceId)
+        {
+            var historyString = await _orchestrationService.GetOrchestrationHistoryAsync(instanceId, null);
+
+            JArray history = JArray.Parse(historyString);
+
+            JArray chronological_history = new JArray(history.OrderBy(obj => (string)obj["TimeStamp"]));
+            foreach (JObject jobj in chronological_history)
+            {
+                var parsed = Enum.TryParse(jobj["EventType"].ToString(), out EventType eventName);
+                jobj["EventType"] = eventName.ToString();
+            }
+
+            ColoredConsole.Write($"History: {chronological_history.ToString(Formatting.Indented)}");
+        }
+
+        public async Task GetRuntimeStatus(string instanceId, bool getAllExecutions)
+        {
+            var statuses = await _client.GetOrchestrationStateAsync(instanceId, allExecutions: getAllExecutions);
+
+            foreach (OrchestrationState status in statuses)
+            {
+                ColoredConsole.WriteLine($"Name: {status.Name}");
+                ColoredConsole.WriteLine($"Instance: {status.OrchestrationInstance}");
+                ColoredConsole.WriteLine($"Version: {status.Version}");
+                ColoredConsole.WriteLine($"TimeCreated: {status.CreatedTime}");
+                ColoredConsole.WriteLine($"CompletedTime: {status.CompletedTime}");
+                ColoredConsole.WriteLine($"LastUpdatedTime: {status.LastUpdatedTime}");
+                ColoredConsole.WriteLine($"Input: {status.Input}");
+                ColoredConsole.WriteLine($"Output: {status.Output}");
+                ColoredConsole.WriteLine($"Status: {status.OrchestrationStatus}");
+            }
+        }
+
+        public async Task RaiseEvent(string instanceId, string eventName, object eventData)
+        {
+            var orchestrationInstance = new OrchestrationInstance
+            {
+                InstanceId = instanceId
+            };
+
+            await _client.RaiseEventAsync(orchestrationInstance, eventName, eventData);
+
+            ColoredConsole.WriteLine(Green($"Raised event {eventName} to instance {instanceId} with data {eventData}"));
```

Comment:
Let's remove the _with data_ part since the event data could be large.
```suggestion
            ColoredConsole.WriteLine(Green($"Raised event {eventName} to instance {instanceId}."));
```